codespan/src/filemap.rs

```diff
@@ -103,8 +103,13 @@ pub struct FileMap {
 }
 
 impl FileMap {
-    /// Construct a new filemap, creating an index of line start locations
-    pub fn new(name: FileName, src: String, start: ByteIndex) -> FileMap {
+    /// Construct a new, standalone filemap. Can be usefol for tests which consists of a single
+    /// source file
```

Comment:
Spelling mistake! 😅  How about:

```rust
/// Construct a new, standalone filemap.
///
/// This can be useful for tests that consist of a single source file. Production code should however 
/// use `CodeMap::add_filemap` or `CodeMap::add_filemap_from_disk` instead.
```

Reply:
Ooops, that is better still though.